lib/ansible/modules/cloud/amazon/aws_kms_info.py

```diff
@@ -360,6 +365,8 @@ def get_key_details(connection, module, key_id, tokens=None):
                          exception=traceback.format_exc(),
                          **camel_dict_to_snake_dict(e.response))
     result['aliases'] = aliases.get(result['KeyId'], [])
+    current_rotation_status = connection.get_key_rotation_status(KeyId=key_id)
```

Comment:
It would be better to use the AWSRetry decorator to wrap this call.  See get_key_policy_with_backoff() for an example.

This ensures that we use a retry + backoff when you hit things like the API rate limits.